packages/fast-components-styles-msft/src/context-menu/index.ts

```diff
@@ -3,17 +3,18 @@ import { ComponentStyles } from "@microsoft/fast-jss-manager";
 import { format, toPx } from "@microsoft/fast-jss-utilities";
 import { DesignSystem } from "../design-system";
 import { applyElevatedCornerRadius } from "../utilities/border";
-import { backgroundColor, designUnit, outlineWidth } from "../utilities/design-system";
+import { designUnit, outlineWidth } from "../utilities/design-system";
 import { applyElevation, ElevationMultiplier } from "../utilities/elevation";
 import {
     HighContrastColor,
     highContrastOptOutProperty,
     highContrastSelector,
 } from "../utilities/high-contrast";
+import { neutralLayerFloating } from "../utilities/color";
 
 const styles: ComponentStyles<ContextMenuClassNameContract, DesignSystem> = {
     contextMenu: {
-        background: backgroundColor,
+        background: neutralLayerFloating,

```

Comment:
I think this can go back to `backgroundColor` since it's wrapped in the correct color <Background> now.

Reply:
This needs to remain since the styles get applied outside the context of the background change.